packages/theme-chalk/src/input.scss

```
@@ -212,6 +212,10 @@
     justify-content: center;
     align-items: center;
     transition: all var(--el-transition-duration);
+
+    &.#{$namespace}-input__icon {
```

Maybe it should be like this:
```suggestion
    &.#{$namespace}-icon {
```